### DISCUSSION TITLE
Alist消耗的系统资源太大

### DISCUSSION THREAD

**Discussion by STEVENTAN100** [AlistGo/alist#3097] [General]
前两天把百度网盘1T多的文件迁移到OneDrive，发现复制目录卡死，拷贝文件也无法取消，而且莫名其妙占据了3G多的硬盘空间，重启后才发现的

**Comment by anwen-anyi** [comment]
你要先了解一下Alist 复制 上传 的运行方式好么，都是先下载到本地/temp文件再上传上去的，下载到本地肯定会占空间 上传完了会自动删除释放出来

**Reply by STEVENTAN100** [reply]
没有，如果是流式上传就不会占用磁盘空间，一般是直接占用内存，我想表达的可能是复制目录的那些已完成不显示会不会好一点？只显示有报错的复制失败的文件。

**Comment by hkiceflow** [comment]
是的，瞬间传输太大会突然卡死，应该是硬盘卡死了，没有做个优化设置最高占用率这种的
